modules/optimization/test/tests/optimizationreporter/bc_load_linearFunction/gold/main_out_OptimizationReporter_0001.csv

```diff
@@ -1,5 +0,0 @@
-grad_wrt_p,measurement_values,measurement_xcoord,measurement_ycoord,measurement_zcoord,misfit_values,p1,p2,simulation_values,measurement_time
-0,207,0.2,0.2,0,0.11599999999999999,113.26514660501002,287.85864907637,207,0
-0,204,0.8,0.6,0,-0.17800000000000002,0.0,0.0,204,0
-0,185,0.2,1.4,0,0.08900000000000001,0.0,0.0,185,0
-0,125,0.8,1.8,0,-0.2,0.0,0.0,125,0
```

Comment:
I added this file to the csvdiff.  I'm using the other file to check that it always converges in a single iteration when using a hessian.  The tolerances are loos because I only want to diff the measurement/simulation values and bcs. Everything else should go to zero.

Reply:
I added this file to the csvdiff.  I'm using the other file to check that it always converges in a single iteration when using a hessian.  The tolerances are loos because I only want to diff the measurement/simulation values and bcs. Everything else should go to zero.